cdap-docs/admin-manual/build.sh

```diff
@@ -20,7 +20,7 @@ source ../vars
 source ../_common/common-build.sh
 
 DEFAULT_XML="../../cdap-common/src/main/resources/cdap-default.xml"
-DEFAULT_XML_MD5_HASH="274486b983a68498c86f7bf02c1fc9a5"
+DEFAULT_XML_MD5_HASH="f673145660df18116877b74baac8c5aa"
```

Comment:
If you make any of the above changes, re-run``./build.sh check`` to generate a new hash.